### PR TITLE
Update hero section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
# Resolves

- [AR-2671](https://team-1624093970686.atlassian.net/browse/AR-2671)

## Changes

- Update primary CTA text (Getting Started), add secondary CTA (Go to docs), change handler method names
- Remove social support icons (Telegram, Discord, Twitter) and their styles
- Remove "Participate in Arcana Aplha testnet launch" chip
- Create a component ArcanaIntroVideo and add youtube embed logic
- Remove glob animation (Node art grid)
- Remove VVideoPlayer, home/Introduction components and remove it's reference from pages/index.vue
- Add circular eclipses and orbs around VVideoThumbnail container
- Remove unused styles

[**Note**]: Exported orbs as png to preserve the glowing gradients. Exporting it to svg removes the orangish glow on the orbs

# Screenshots

Screenshot from 2022-06-03 18-57-44
Screenshot from 2022-06-03 18-58-02

# Checklist

- [x] The branch name follows the format: `developer-name/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: components/home/ArcanaIntroVideo.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  emits: ['close'],
+}
+</script>
+
+<style scoped lang="postcss">
+@import url('../lib/media-query-helper.css');
+
+.youtube-embed {
+  width: 64rem;
+  height: 36rem;
```
Comment (ajithranka) [inline components/home/ArcanaIntroVideo.vue:32]: Same width and height, but I find this easier to understand:

```suggestion
  width: 64rem;
  height: calc(64rem / 1.77);
```

**Review by ajithranka — COMMENTED** [review]
- Recheck the file name casing for the video thumbnail image.
- We get this layout shift on loading the page because the thumbnail image takes time to load. See if adding `rel="preload"` to the `img` helps, else we'll need to explore other solutions for preloading the images.

https://user-images.githubusercontent.com/11095038/172096754-afe3fa93-05cf-4840-93df-7daff384933f.mov

File: components/home/ArcanaIntroVideo.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-overlay>
+    <iframe
+      src="https://www.youtube.com/embed/itP6iLLskro?autoplay=1"
```
[the remark this answers is not in the document]
Comment (shrinathprabhu) [reply components/home/ArcanaIntroVideo.vue:4]: Resolved at https://github.com/arcana-network/website/pull/97/commits/693ec796b541c985ab953f96dfa0141e6cb87795

File: components/home/ArcanaIntroVideo.vue (hunk 1 above)
Comment (shrinathprabhu) [reply components/home/ArcanaIntroVideo.vue]: Sounds good. Resolved at https://github.com/arcana-network/website/pull/97/commits/79ab2fb2b7c95beebb50101aaab4dd86eca76084

**Review by ajithranka — APPROVED** [review]

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**